### DISCUSSION TITLE
Other multimedia keys?

### DISCUSSION THREAD

**Discussion by gregmo72** [kriomant/ch57x-keyboard-tool#92] [Q&A]
Does anyone know if there is a way to use multimedia keys other than those specified in "show-keys"?  This is the second keyboard I bought hoping to be able to program to send "rewind" and "fast forward" and so far I'm lost.  Under show-keys it mentions using a decimal code but not much else.  I searched trying to find a list of decimal codes that I might put here but no luck so far.  

TIA

**Comment by kriomant** [comment]
You may find all HID codes for USB devices here: https://www.usb.org/sites/default/files/documents/hut1_12v2.pdf (section 10).
If this doesn't work, just assign some combination like 'cmd-ctrl-alt-g' to key and remap it to do arbitrary action using some third-party software like BetterTouchTool.

**Comment by gregmo72** [comment]
That did point me in the right direction, thanks.  Funny thing, that list stops listing at decimal code 231 (shows the rest of the range as reserved) but the multimedia codes don't start until 266.  I've not tested it yet, but it looks like the codes I am looking for are 275 and 276 for KEY_MEDIA_BACK and KEY_MEDIA_FORWARD.
